api/util/GenerateTokenScript.js

```diff
@@ -0,0 +1,104 @@
+const fs = require('fs');
+const readline = require('readline');
+const { google } = require('googleapis');
+
+/**
+ * Prompts user for client ID & secret and writes them to config
+ * file containing auth credentials, then generates a token.json file for GCP
+ * @param {String} configPath the directory path of the config.json file
+ */
+const setAuthCredentials = async (configPath) => {
+  const rawConfig = fs.readFileSync(configPath);
+  console.debug('Welcome to SCE\'s Google Cloud API Token Generator!');
+
+  if(rawConfig) {
+    const clientID = await inputPrompt('Please enter the Client ID: ');
+    const clientSecret = await inputPrompt('Please enter the Client Secret: ');
+
+    const config = JSON.parse(rawConfig);
+    if(clientID) {
+      config.googleApiKeys.CLIENT_ID = clientID;
+    }
+    if(clientSecret) {
+      config.googleApiKeys.CLIENT_SECRET = clientSecret;
+    }
+
+    fs.writeFile(configPath, JSON.stringify(config), (err) => {
+      if(err) return console.debug('ERR', err);
```

Comment:
consider adding colors to print statements [like this](https://github.com/SCE-Development/Core-v4/blob/dev/api/util/CreateUserScript.js#L9)

Reply:
Add a better error message here, e.g.
```suggestion
      if(err) {
        return console.debug(`A problem occurred trying to write to ${configPath}`, err);
      }
```